lib/sidekiq/instrument/worker.rb

```diff
@@ -21,18 +21,22 @@ def perform
       self.class::METRIC_NAMES.each do |method, stat|
         stat ||= method
 
-        StatsD.gauge "shared.sidekiq.stats.#{stat}", info.send(method)
+        StatsD.gauge("shared.sidekiq.stats.#{stat}", info.send(method))
+        DogStatsD.gauge("shared.sidekiq.stats.#{stat}", info.send(method), tags: ['sidekiq'])
       end
 
       working = Sidekiq::ProcessSet.new.select { |p| p[:busy] == 1 }.count
-      StatsD.gauge "shared.sidekiq.stats.working", working
+      StatsD.gauge("shared.sidekiq.stats.working", working)
+      DogStatsD.gauge("shared.sidekiq.stats.working", working, tags: ['sidekiq'])
 
       info.queues.each do |name, size|
-        StatsD.gauge "shared.sidekiq.#{name}.size", size
+        StatsD.gauge("shared.sidekiq.#{name}.size", size)
+        DogStatsD.gauge("shared.sidekiq.#{name}.size", size, tags: ['sidekiq'])
       end
 
       Sidekiq::Queue.all.each do |queue|
-        StatsD.gauge "shared.sidekiq.#{queue.name}.latency", queue.latency
+        StatsD.gauge("shared.sidekiq.#{queue.name}.latency", queue.latency)
+        DogStatsD.gauge("shared.sidekiq.#{queue.name}.latency", queue.latency, tags: ['sidekiq'])
```

Comment:
For DogStatsD, we follow the convention that things like `queue.name` should be passed as a tag. This should allow for more data aggregation on the metric, filtering on the queue name or other tags such as environment or host. The metric itself should be relatively generic. For a brief overview, check out [Getting Started with Tags from DogStatsD](https://docs.datadoghq.com/getting_started/tagging/)

Reply:
+1. 

Also read [DogStatsD Metrics Submission](https://docs.datadoghq.com/metrics/custom_metrics/dogstatsd_metrics_submission/)